### PR TITLE
add custom background

### PR DESCRIPTION
**Pull request by sarveshpro**
Changes in this pull request:
- add new tenant treecounter theme - forest

this is for klumforest.com but we are making this theme available for public.

Todo

- [x] Forest name hides target

Some example screen shots (having the wrong tenant, so the tree count is wrong):

Bildschirmfoto 2021-05-26 um 10 06 22Bildschirmfoto 2021-05-26 um 10 06 42

### CODE REVIEW

File: src/TreeTenantCounter/App.svelte [hunk 1]
```diff
@@ -66,7 +91,7 @@
               {language[locale].treesPlanted}
             </p>
           </div>
-          {#if goal != 0}
+          {#if goal && goal != "" && goal != 0}
```
Comment (norbertschuler) [inline src/TreeTenantCounter/App.svelte:94]: Are these additional checks necessary?

File: public/data/locales/en.json [hunk 2]
```diff
@@ -10,5 +10,7 @@
     "forestfrontrunners": "Forest Frontrunners",
     "mostrecent": "MOST RECENT",
     "mosttrees": "MOST TREES",
-    "trees": "Trees"
+    "trees": "Trees",
+    "the": "The",
+    "forestGrows": "Forest Grows."
```
Comment (norbertschuler) [inline public/data/locales/en.json:15]: This won't work in every language. And I am also not sure if capitalizing every word looks good in German, e.g. 'Der Klum Wald Wächst' - for me it looks unreadable not given the fact that is has a mistake and in correct grammar we would have to write "Klum-Wald".
We should use a i18n resource with a placeholder instead for flexibility for every language.

**Review by norbertschuler — DISMISSED** [review]

File: public/data/locales/en.json (hunk 2 above)
Comment (norbertschuler) [reply public/data/locales/en.json:15]: Btw. we have not configured LingoHub for this project and need to add the German translation manually before merging!

Comment (norbertschuler) [reply public/data/locales/en.json:15]: As adding i18n to this project would increase the project, we cannot use a placeholder solution. So I added (wrong) German translations as good as possible. now.

File: src/TreeTenantCounter/App.svelte [hunk 3]
```diff
@@ -75,7 +100,8 @@
             </div>
           {/if}
         </div>
-
+        {/if}
+        {#if goal && goal != "" && goal != 0}
```
Comment (norbertschuler) [inline src/TreeTenantCounter/App.svelte:104]: Same as above. Is `{#if goal}` not enough to check?

Comment (rishsingh07) [reply src/TreeTenantCounter/App.svelte:104]: @sarveshpro and I tried to solve that with minimal code but only this satisfied every condition

Comment (norbertschuler) [reply src/TreeTenantCounter/App.svelte:104]: Well, I couldn't reproduce errors using `{#if goal}`, but better be safe than sorry :-)

File: src/TreeTenantCounter/App.svelte (hunk 1 above)
Comment (norbertschuler) [reply src/TreeTenantCounter/App.svelte:94]: see discussion below

**Review by norbertschuler — APPROVED** [review]

**Pull request merged**